test/GithubApi.Redirect.test.js

```diff
@@ -0,0 +1,15 @@
+const { expect } = require('chai');
+const axios = require('axios');
+const chaiSubset = require('chai-subset');
+const chai = require('chai');
+
+chai.use(chaiSubset);
+const object = axios.create({
+  headers: {
+    Authorization: `token ${process.env.ACCESS_TOKEN}`
+  }
+});
+it('test', async () => {
```

Comment:
you need to add an additional `it` to verify the redirect is also present when you do a call with the head method, and the two `it`s should go within a `describe`

Reply:
done @holgiosalos